DOCSP-30347: Convert Manage Users landing page (#3021)
mongodb/docs-realm: source/sdk/kotlin/users/delete-users.txt

   :depth: 2
   :class: singlecol

This page describes how to remove and delete authenticated users from 

> your Atlas App Services App. Learn how to :ref:`<kotlin-authenticate-users>`.

Reviewer: ```suggestion
your Atlas App Services App. For more information on how to create and authenticate users, see :ref:`<kotlin-authenticate-users>`.
```
Did you mean to do something like this to introduce further reading?

Remove a User
-------------

To remove a ``User`` object from your client app, call the `remove 
<{+kotlin-sync-prefix+}io.realm.kotlin.mongodb/-user/remove.html>`__
method on a user: 

.. literalinclude:: /examples/generated/kotlin/AuthenticationTest.snippet.remove-user.kt
   :language: kotlin

> The ``user.remove()`` method: 

Reviewer: ```suggestion
The ``user.remove()`` method acts as follows: 
```
Style Guide [Lists section](https://www.mongodb.com/docs/meta/style-guide/style/lists/) says lists should start with capital letter. Reformatting to help.

> - *does not* delete the ``User`` object from the Atlas App Services App

Reviewer: ```suggestion
- *Does not* delete the ``User`` object from the Atlas App Services App.
```

> - :ref:`logs the user <kotlin-logout>` out if they are logged in

Reviewer: ```suggestion
- :ref:`Logs the user <kotlin-logout>` out if they are logged in.
```

> - deletes synced realms associated with the user from the device

Reviewer: ```suggestion
- Deletes synced realms associated with the user from the device.
```

> - sets the

Reviewer: ```suggestion
- Sets the
```

  `User.State <{+kotlin-sync-prefix+}io.realm.kotlin.mongodb/-user/-state/index.html>`__ 

>   to ``REMOVED``

Reviewer: ```suggestion
  to ``REMOVED``.
```

Because removing a user deletes any synced realms owned by the user, you 
should only call this method after closing the user's realms.

Delete a User
-------------

To permanently delete a ``User`` object from both your client app 
*and* the Atlas App Services App, call the `delete 
<{+kotlin-sync-prefix+}io.realm.kotlin.mongodb/-user/delete.html>`__ 
method on a logged-in user:

.. literalinclude:: /examples/generated/kotlin/AuthenticationTest.snippet.delete-user.kt
   :language: kotlin

> The ``user.delete()`` method:

Reviewer: ```suggestion
The ``user.delete()`` method performs the following:
```
Same as above comment, lists should start with capital. I believe should end in period.

> - deletes synced realms associated with the user from the device

Reviewer: ```suggestion
- Deletes synced realms associated with the user from the device.
```
Same as in above section, capitalize first word of each item in list and add period.
